lib/validations/etl.js

```diff
@@ -162,7 +183,7 @@ function start() {
 // refresh connections
 // every minute
 setInterval(start, 60000);
```

Comment:
I think this and validations.start should be moved inside the then after manifests.start